src/modules/rust.rs

```diff
@@ -127,24 +127,37 @@ fn find_rust_toolchain_file(context: &Context) -> Option<String> {
     // Look for 'rust-toolchain' as rustup does.
     // https://github.com/rust-lang/rustup.rs/blob/d84e6e50126bccd84649e42482fc35a11d019401/src/config.rs#L320-L358
```

Comment:
Please update this comment to match a commit with the newer behaviour.

Reply:
Missed that, will update it.